plugin.py

```diff
@@ -144,3 +145,35 @@ def on_response_async(self, response: Any) -> None:
 
     def on_error_async(self, error: Any) -> None:
         pass
+
+
+class LspTexlabCleanAuxiliaryCommand(LspTextCommand):
+
+    session_name = PLUGIN_NAME
+
+    def run(self, _: sublime.Edit) -> None:
+        session = self.session_by_name(PLUGIN_NAME)
+        if not session:
+            return
+
+        execute_command_params = {
+            "command": "texlab.cleanAuxiliary",
+            "arguments": [text_document_identifier(self.view)]
+        }  # type: ExecuteCommandParams
+        session.execute_command(execute_command_params, False)
+
+
+class LspTexlabCleanArtifactsCommand(LspTextCommand):
+
+    session_name = PLUGIN_NAME
+
+    def run(self, _: sublime.Edit) -> None:
+        session = self.session_by_name(PLUGIN_NAME)
+        if not session:
+            return
+
+        execute_command_params = {
+            "command": "texlab.cleanArtifacts",
+            "arguments": [text_document_identifier(self.view)]
+        }  # type: ExecuteCommandParams
+        session.execute_command(execute_command_params, False)
```

Comment:
I believe that those can be handled by the built-in `lsp_execute` command - https://lsp.sublimetext.io/commands/#execute-server-commands

Reply:
That is a great solution. Did not know about `lsp_execute`. Thanks a lot for your input!